config/config.go

```diff
@@ -50,9 +51,9 @@ func LoadConfigFile(path string) (*Config, error) {
 func LoadConfigDev() (*Config, chan struct{}, string, error) {
 	// start a vault dev instance
 	shutdownCh := initDevVaultCore()
```

Comment:
`initDevVaultCore()` starts a goroutine that will take care of launching and keeping dev vault alive